manual/development-process/production-deployment.md

```diff
@@ -0,0 +1,178 @@
+same time as theirs, and there failures, it will be harder for them to debug.
+
+We have agreed to some rules about when it is OK to deploy to Production:
+
+- Only when you have time to monitor the deployment. Decide how long you need based on the size and complexity of the
+  change.
+- Only when there are other TDR developers around. We are a small team with flexible working hours, so there are no
+  restrictions based on the time of day, but you should make sure that at least one other developer is around to help
+  debug any deployment issues.
+- Not on a Friday or the day before a bank holiday. Ideally, we would also deploy on Fridays to avoid a build-up of
+  changes that would make the deployment larger, but we need to demonstrate that our deployments are low-risk and that
+  we can catch issues early with tests and monitoring.
+- If the end-to-end tests passed on Staging after the most recent deployment
+
+These rules only apply to Production, but it's usually better to apply them to Staging as well. This means that you can
+deploy to Production shortly after deploying to Staging, which prevents coordination problems where changes are deployed
+to Production in the wrong order.
+
+## How to deploy a change to Production
+
+The developer who made a change is responsible for the deployment. If you do not have permission to deploy to Staging
+and Production, you should ask a developer with Production access to run the actual deployments, but you yourself should
+still follow the steps below to make sure everything is working at each step.
+
+This guide explains how to deploy a single service. If you need to deploy multiple changes (e.g. a database, API and
+frontend change), you should follow these steps separately for each service. Each step needs to be compatible with the
+other services currently deployed to each environment. For example, you should not deploy an API change which depends on
+a database change that hasn't been deployed yet. The end-to-end tests should help you catch problems like this so that
+you can roll out the change without any downtime.
+
+**Always deploy to each environment in order.** Deployment to Integration should be first, followed by Staging, and
+finally Production.
+
+### Step 1: Merge your branch
+
+This assumes that you have created a pull request for your changes in GitHub, and it has been approved by a reviewer.
+TDR repos are set up so that you can only merge a branch once it has been approved and the CI checks have succeeded (see
+the [repo setup guide]).
+
+### Step 2: Deploy to Integration
+
+Most repos, including the main codebases like the frontend and consignment API, are deployed to Integration
+automatically after the main branch build has finished. If you are unsure, check the Jenkinsfile in the repo.
+
+Some repos, like the [database migrations], have a Jenkins deployment job that needs to be started manually. Check the
+repo's Readme for instructions.
```

Comment:
just a minor change (ignore if it's not worth it): "README" instead of "Readme"

Reply:
I never know what capitalization to use for that word. 🤷‍♀️ I've left it as Readme so it looks LESS LIKE SHOUTING but I don't really mind.